packages/salesforcedx-vscode-core/src/commands/auth/forceAuthDevHub.ts

```diff
@@ -58,17 +60,20 @@ export class ForceAuthDevHubContainerExecutor extends ForceAuthWebLoginContainer
   }
 }
 
+export const DEFAULT_ALIAS = 'vscodeOrg';
```

Comment:
can you update this to use: https://github.com/forcedotcom/salesforcedx-vscode/blob/96fe184c8440c206ddb4bc004387eb17c454db77/packages/salesforcedx-vscode-core/src/commands/auth/authParamsGatherer.ts#L18